cronex/__init__.py

```diff
@@ -256,9 +261,17 @@ def check_trigger(self, date_tuple, utc_offset=0):
                 # None of the expressions matched which means this field fails
                 return False
 
-        # Arriving at this point means the date landed within the constraints
-        # of all fields; the associated trigger should be fired.
-        return True
+        is_wild = self.string_tab[0] == '*' or self.string_tab[1] == '*'
```

Comment:
Overall I find the change kind of peculiar. Shoving half the logic needed to achieve the skew handling into cronex then expecting the library users to roll their own wrapper feels wrong to me. I think that either cronex should support skewing as a generic feature or it should expose enough state for the caller to decide whether or not to ignore a trigger without changing the existing interface. I need to give this implementation some more though.

Reply:
It's a fair criticism. I was trying to make the most minimal change I could that would solve the use case.